python/metatensor-operations/metatensor/operations/_backend.py

```diff
@@ -16,12 +16,17 @@ def torch_jit_is_scripting():
     return False
 
 
+def torch_jit_script(function):
+    return function
```

Comment:
Maybe we should a small comment why we need this function. On a first glance this just looks like a useless unity operation.

Reply:
Good point